build-tools/xa-prep-tasks/Xamarin.Android.BuildTools.PrepTasks/ProcessMSBuildTiming.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Xml.Linq;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+
+namespace Xamarin.Android.BuildTools.PrepTasks
+{
+	public class ProcessMSBuildTiming : Task
```

Comment:
Some kind of comment description of *what this does* might be useful. :-)

Reply:
I suspect this should go into `Xamarin.Android.Tools.BootstrapTasks`, not `xa-prep-tasks`. `xa-prep-tasks` is the "bare minimum" needed to prepare the entire repo, meaning external dependencies are potentially problematic. (This is why `xa-prep-tasks` doesn't have a dependency on `libZipSharp`, for example.)